graphrag/index/run/profiling.py

```diff
@@ -69,7 +69,3 @@ async def _write_workflow_stats(
         await _save_profiler_stats(
             storage, workflow.name, workflow_result.memory_profile
         )
-
-    log.debug(
```

Comment:
Why are we removing this from profiling?

Reply:
I already checked this with @natoverse yesterday, because now the pipeline supports empty dataframes.

Reply:
@AlonsoGuevara this call was just a debug, which now fails because it assumes a dataframe with at least one row. The call to the profiler happens above anyway. We can put in a "null check" if you think it's really helpful